includes/wporg-meeting-posttype.php

```diff
@@ -864,7 +866,7 @@ public function meeting_time_shortcode( $attr, $content = '' ) {
 				$out .= '<time class="date" date-time="' . esc_attr( $next_meeting_iso ) . '" title="' . esc_attr( $next_meeting_iso ) . '">' . $next_meeting_display . '</time> ';
 				$out .= sprintf( esc_html__( '(%s from now)', 'wporg-meeting-calendar' ), human_time_diff( $next_meeting_timestamp, current_time( 'timestamp' ) ) );
 				if ( $post->location && $slack_channel ) {
-					$out .= ' ' . sprintf( wp_kses( __( 'at <a href="%1$s">%2$s</a> on Slack', 'wporg-meeting-calendar' ), array( 'a' => array( 'href' => array() ) ) ), 'https://wordpress.slack.com/messages/' . $slack_channel, $post->location );
+					$out .= ' ' . sprintf( wp_kses( __( 'accessible via <a href="%1$s">%2$s</a> on Slack or  <a href="%3$s">%4$s:community.wordpress.org</a> on Matrix', 'wporg-meeting-calendar' ), array( 'a' => array( 'href' => array() ) ) ), 'https://wordpress.slack.com/messages/' . $slack_channel, $post->location, '/' . strtolower( 'Core Performance' === $attr['team'] ? 'performance' : $attr['team'] ) . '/chat/', $post->location );
```

Comment:
This change comprises to commits made directly in SVN and not synced back to GitHub. See https://dotorg.trac.wordpress.org/browser/wordpress/website/wp-content/plugins/wporg-meeting-calendar/includes/wporg-meeting-posttype.php?rev=20906 and https://dotorg.trac.wordpress.org/browser/wordpress/website/wp-content/plugins/wporg-meeting-calendar/includes/wporg-meeting-posttype.php?rev=21316